content/en/integrations/guide/data-center-migration.md

```diff
@@ -0,0 +1,158 @@
+
+## Migrate and synchronize Datadog resources across organizations
+
+<div class="alert alert-info">This feature is in beta.</div>
+
+[`datadog-sync-cli`][1] is a command line tool that you can use to synchronize resources, specify resources to migrate, and add filters to selectively migrate resources based on resource attributes. 
+
+When you run the `import` command, the package reads the items from the specified resources in the source organization and saves them locally as JSON files in a `resources/source` folder. After running the `sync` command, the package uses the local cache (unless a `--force-missing-dependencies` flag is passed) to create the resources in the destination organization and saves what was pushed locally. 
+
+You need to specify the API and application keys for a source organization and a destination organization as individual parameters. For more information about commands, see [Usage][10].
+
+### Prerequisites
+
+The `datadog-sync-cli` package requires Python 3 >= 3.9.
+
+### Resource attributes
+
+A [Datadog resource][5] is an element or tool that you use to monitor your data in Datadog, such as a dashboard, monitor, or a notebook. 
+
+You can use `datadog-sync-cli` to migrate the following resources:
+
+{{% datadog-sync-resources %}}
+
+### Dependent resources
+
+Many Datadog resources are interdependent, meaning that a resource can reference many resources such as `roles` and `dashboard`, which may include widgets that use Datadog monitor or Synthetic test data. 
+
+While the `datadog-sync-cli` package synchronizes these resources to ensure dependencies are not broken, you must ensure that dependent resources are also imported and synced if you are importing and syncing a _subset_ of resources.
+
+{{% datadog-sync-resources-dependencies %}}
+
+At this point, you have a destination organization in a new data center which possesses a list of Datadog resources migrated from your old Datadog account. These resources are empty because they do not contain any monitoring data.
+
+### Limitations
+
+The `datadog-sync-cli` tool does not migrate intake data such as ingested logs or metrics, and does not migrate API keys or integrations.
+
+The following Datadog resources are not covered by the `datadog-sync-cli` tool:
+
+Log Management
+: Saved views, log facets, log JSON pre-processing attributes, and any increased log monitor quota (the default number of log monitors allowed is 1000). </br></br> To migrate existing logs to a new Datadog account, use [Log Archiving][2] and [Log Rehydration™][3].
+
+### Migrate integrations and real-time data
+
+Integrations
+: Migrating to another data center requires you to re-add your integration. Use a CI mechanism to mass update agents (by using the `logs_pipelines` resource).
+
+APM
+: Setting a second primary tag in APM requires you to update existing dashboards and monitor queries.
```

Comment:
```suggestion
: Setting a second primary tag in APM requires you to update existing dashboards and monitor queries.
```

Is this relevant for migrating data centers? Do we expect customers to add a second primary tag during migration?